src/components/AgreementSections/CoveredEResourcesList.js

```diff
@@ -201,6 +209,41 @@ export default class CoveredEResourcesList extends React.Component {
     );
   }
 
+  renderPtiUrl = (eresource) => {
+    const url = eresource?._object?.pti?.url;
+    return url ? (
+      <Tooltip
+        key={eresource.id}
+        id={`tooltip-${eresource.id}`}
+        placement="bottom"
+        text={<FormattedMessage
+          id="ui-agreements.eresources.accessTitleOnPlatform"
+          values={{
+            name: eresource?._object?.pti?.name
+          }}
+        />}
+      >
+        {({ ref, ariaIds }) => (
+          <div
+            ref={ref}
+            aria-labelledby={ariaIds.text}
+          >
+            <a
+              href={url}
+              onClick={e => e.stopPropagation()}
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              <FormattedMessage id="ui-agreements.eresources.titleOnPlatform" />
```

Comment:
Believe the preferred way of doing this is `<Icon><FormattedMessage /></Icon>` so that the icon is vertically-aligned correctly.